deps/src/greaseLib/logger.cc

```diff
@@ -888,6 +888,7 @@ void GreaseLogger::_doLibCallback(GreaseLogger::logTarget::writeCBData &data) {
 		buf->_id = (int) data.t->myId;
 		buf->_shadow = data.b; // hide the original object here in void *
 		data.t->logCallback(NULL,buf,data.t->myId);
+		LFREE(buf);
```

Comment:
What's LFREE?

Reply:
Possibly a wrapper to make it OS agnostic. Since allocation in the file is using LMALLOC / using similar wrapper for free - LFREE
https://github.com/armPelionEdge/greasego/pull/1/files/386a3dcd165a2fd03f71661fcb46f1903c44fa5e#diff-c1f91c9316238b8b429f0d2db93cb350R885 --> https://github.com/armPelionEdge/greasego/blob/5b578dd9578b693c0938392a8cb4efbe76ec3918/deps/src/greaseLib/grease_lib.cc#L75